README.md

```diff
@@ -42,4 +42,17 @@ gradle clean bootrun
 - Type this url address:
 ```
 http://localhost:9091/api/v1/swagger-ui.html
+```
+## To execute MOI whit MySql DataBase
+- Install Docker:
+```
+https://hub.docker.com/editions/community/docker-ce-desktop-windows/
+```
+- Open CMD console and execute:
+```
+docker version
+```
```

Comment:
It would be better if you put an example of result expected when execute some command.